.github/workflows/maven-graalvm.yml

```diff
@@ -0,0 +1,37 @@
+name: GraalVM Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        os: [ubuntu-24.04, macOS-13]
+        java: [ '21', '25' ]
+        distribution: [ 'graalvm-community' ]
+        maven: ['3.9.11']
+      fail-fast: false
+    name: ${{ matrix.os }} JDK ${{ matrix.java }}.${{ matrix.distribution }}
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+      - name: Set up GraalVM
+        uses: graalvm/setup-graalvm@v1
+        with:
+          distribution: ${{ matrix.distribution }}
+          java-version: ${{ matrix.java }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v4.6
```

Comment:
`v4.6` doesn't seem to exist. Can we try `v5`?

Reply:
Yes that should work 
```
- name: Set up Maven
      uses: stCarolas/setup-maven@v5
      with:
        maven-version: 3.8.2
```
https://github.com/stCarolas/setup-maven will change that